pages/01_embedding_and_jsapi.md

```diff
@@ -7,11 +7,38 @@ The easiest method for embedding a Tableau view (dashboard or visualization) is
 ![Embed Code]({{ site.baseurl }}/img/embed_code.png)
 
 Once acquired, this code can be pasted into HTML.
-This method is useful for simple embedding, such as embedding into blogs or internal knowledge bases, but **only very simple embedding scenarios should use the embed code. Most deployments should instead use the JavaScript API.** It is not significantly more work to embed with the JavaScript API and doing so will gain you flexibility and power in your embedded deployment.
+This method is useful for simple embedding, such as embedding into blogs or internal knowledge bases, but **only very simple embedding scenarios should use the embed code. Most deployments should instead use the JavaScript API or the Embedding API v3.** It is not significantly more work to embed with the JavaScript API or the Embedding API v3, and doing so will gain you flexibility and power in your embedded deployment.
+
+## Embedding with the Embedding API v3
+
+The Embedding API v3 is the newest and easiest way to embed Tableau views, and provides you with options for enhancements, including improved security and integration for Connected Apps and external authorization servers (EAS). Available starting with Tableau 2021.4, you can use the Embedding API v3 to add a Tableau view as a web component in your HTML pages. Using the Tableau viz web component, you can complete your basic embed scenarios without having to write JavaScript code. The Embedding API v3 is currently available now for basic embedding. If you are looking to replace your embed code, or starting a new embedding project, you should consider using the Embedding API v3. The Embedding API v3 is under active development and in future releases will provide full parity with the JavaScript API, giving you a rich set of features that you can access in your JavaScript or TypeScript code.
```

Comment:
Probably add that is the new, modern replacement for our JavaScript API v2. It might not be clear for the readers that Embedding v3 is the new version of the JavaScript API v2

Reply:
Make sense. I'll add that.  I wasn't sure how much to go into that since we aren't at parity yet.